akka-stream/src/main/scala/akka/stream/javadsl/SubFlow.scala

```diff
@@ -1151,6 +1151,22 @@ class SubFlow[-In, +Out, +Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Flo
     new SubFlow(delegate.alsoTo(that))
 
   /**
+   * Attaches the given [[Sink]] to this [[Flow]] as a wire tap, meaning that elements that pass
+   * through will also be sent to the [[Sink]], without the latter affecting the mainline flow.
```

Comment:
Should also explain the part about it dropping elements if there is backpressure on the tap

Reply:
Presumably this applies to all similar comments, right?

Reply:
Correct!